src/portserver.py

```diff
@@ -51,11 +53,32 @@ def _get_process_command_line(pid):
 
 
 def _get_process_start_time(pid):
-    try:
-        with open('/proc/{}/stat'.format(pid), 'rt') as pid_stat_f:
-            return int(pid_stat_f.readline().split()[21])
-    except IOError:
-        return 0
+    if sys.platform == 'win32':
+        cmd_result = subprocess.run(
+            f'wmic process where ProcessID="{pid}" get CreationDate',
```

Comment:
Could `int(psutil.Process(pid).create_time())` perhaps do this dance for us?  cross platform even, meaning we wouldn't need our platform specific code...